### DISCUSSION TITLE
Sensor ignore backill

### DISCUSSION THREAD

**Discussion by Max0u** [dagster-io/dagster#23072] [Q&A]
Is there a way to know if the cursor status comes from a backfill? We would like to ignore and avoid returning Runrequest on backfill

**Comment by garethbrickman** [comment]
Runs that were triggered as part of a backfill will be assigned a backfill tag that includes the backfill ID. Check out the answer in this discussion for examples how to check for that: https://github.com/dagster-io/dagster/discussions/16189
